core-domain/src/test/java/org/uniprot/core/uniref/impl/UniRefEntryLightImplTest.java

```diff
@@ -16,7 +16,6 @@ void needDefaultConstructorForJsonDeserialization() {
         assertNotNull(obj);
```

Comment:
It is better if we have few more tests like verifying the value of commonTaxon and organisms

Reply:
We cover it in the builder test